output/transform/postgres.go

```diff
@@ -164,6 +164,8 @@ func transformPostgresServerStats(s snapshot.FullSnapshot, transientState state.
 		XminHorizonReplicationSlotCatalog: transientState.ServerStats.FullXminHorizonReplicationSlotCatalog(),
 		XminHorizonPreparedXact:           transientState.ServerStats.FullXminHorizonPreparedXact(),
 		XminHorizonStandby:                transientState.ServerStats.FullXminHorizonStandby(),
+		PgStatStatementsDealloc:           transientState.ServerStats.PgStatStatementsDealloc,
```

Comment:
I think we should run our usual diffing logic for the `dealloc` field, so we don't send the absolute value, but rather a diff to the previous one - that may seem unnecessarily complex to implement, but I think its necessary to be able to answer "did any deallocations occur in this timeframe" (e.g. when investigating missing query texts for workloads at a given time)

Reply:
Ah, totally forgot about doing the diff-ing, will do 👍

Reply:
This is addressed, now it's sending a diff-ed version